lib/vtls/darwinssl.c

```diff
@@ -2573,6 +2661,15 @@ void Curl_darwinssl_md5sum(unsigned char *tmp, /* input */
   (void)CC_MD5(tmp, (CC_LONG)tmplen, md5sum);
 }
 
+void Curl_darwinssl_sha256sum(unsigned char *tmp, /* input */
+                           size_t tmplen,
+                           unsigned char *sha25sum, /* output */
+                           size_t sha25len)
+{
+  (void)sha25len;
```

Comment:
This is neither here nor there, but should we validate that the size of the buffer is actually large enough to hold a SHA256 digest? Using `assert` would likely be sufficient.